### ISSUE TITLE
More Robust Dashboard that could be hooked into hapi-dashboard

### ISSUE THREAD

**Issue by ecwillis** [firstandthird/hapi-agenda#21]
Convert the current json api trigger, or write another parallel handler that would provide administrative or at least html views of the jobs instead of a strict API. You could enable, disable, delete jobs that we're initialized at startup.

**Comment by jgallen23** [comment]
yep!  Great idea.  I think it might be completely separate from hapi-dashboard since that was made to be read-only.  The main question is do we include in this repo or have a separate plugin for the admin tool?

**Comment by ecwillis** [comment]
I actually think a separate repo to provide separation of concerns... and we could migrate the JSON api over to there? that way this repo would handle the hapi-agenda plugin code, and the other would provide both API and web views for controlling it.

**Comment by jgallen23** [comment]
yeah, I agree to separate the json api over there too.  Lets talk through this this week
